extensions/config-yaml/deployment/src/test/resources/configprops.yaml

```diff
@@ -4,34 +4,4 @@ sql:
   name: defaultName
   user: defaultUser
   password: defaultPassword
-  type: DEFAULT_TYPE
-  inputs:
```

Comment:
Support for list of objects in YAML was removed in https://github.com/quarkusio/quarkus/commit/ab40883447ea89ae4334c371c315b954ba5f7b5a.